src/common/pure/CurrencySelectButton/styled.tsx

```diff
@@ -9,12 +9,14 @@ export const CurrencySelectWrapper = styled.button<{ isLoading: boolean; $stubbe
   align-items: center;
   cursor: pointer;
   gap: 6px;
-  border: 0;
+  border: ${({ theme }) => (theme.darkMode ? `1px solid ${transparentize(0.5, theme.grey1)}` : 0)};
```

Comment:
Is 0 valid here or should it be `none` ?

Reply:
https://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0